test/helpers/GameFlowWrapper.js

```diff
@@ -56,6 +56,7 @@ class GameFlowWrapper {
     resourceAnyTwo() {
         this.guardCurrentPhase('setup');
         this.allPlayersInInitiativeOrder().forEach((player) => player.clickAnyOfSelectableCards(2));
+        this.allPlayersInInitiativeOrder().forEach((player) => player.clickPrompt('Done'));
```

Comment:
Similar comment as above - let's just roll these two into one `for` loop

Reply:
yessir!